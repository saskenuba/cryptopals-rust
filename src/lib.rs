pub mod b64;
pub mod hex;
pub mod xor;

pub mod challenges;

pub(crate) type AnyResult<T> = anyhow::Result<T>;

static B64: &str = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
