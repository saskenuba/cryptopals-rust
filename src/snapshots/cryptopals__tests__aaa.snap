---
source: src/lib.rs
expression: hex_str.into_b64_string()
---
SSdtIGtpbGxpbmcgeW91ciBicmFpbiBsaWtlIGEgcG9pc29ub3VzIG11c2hyb29t
